Examples/ldap.md

```diff
@@ -0,0 +1,56 @@
+The following example `docker-compose.yml` file shows the setup with LDAP. However, it maybe unclear to a new user what LDAP is and why should one use it. What is the context for it to be useful? What other services should be setup together? Contributions are needed here.
```

Comment:
Like written before I would not put questions that need answers into the readme.